lib/addons/core-addon/index.ts

```diff
@@ -126,4 +139,15 @@ export class CoreAddOn implements ClusterAddOn {
         return result;
     }
 
-}
\ No newline at end of file
+    provideVersion(clusterInfo: ClusterInfo, versionMap: Map<KubernetesVersion, string>) : string {
+        let version: string;
+        if(!versionMap.get(clusterInfo.version)) {
```

Comment:
nitpick: no need to get from the map twice on line 144 and 147. 
`let version = versionMap.get();`

Reply:
resolved